Makefile

```diff
@@ -1,5 +1,44 @@
 DOT := $(shell command -v dot 2> /dev/null)
 
+RAS_REPOS="ras-party" "ras-secure-message" "ras-frontstage" "ras-collection-instrument" "django-oauth2-test" "ras-rm-auth-service" "respondent-home-ui" "response-operations-ui" "response-operations-social-ui" "sdc-uaa"
+RM_REPOS="rm-sample-service" "rm-case-service" "rm-action-service" "rm-actionexporter-service" "iac-service" "rm-sdx-gateway" "rm-collection-exercise-service" "rm-survey-service" "rm-notify-gateway" "rm-comms-template-service" "rm-reporting"
+REPOS=${RAS_REPOS} ${RM_REPOS}
+REPOS=${RAS_REPOS} ${RM_REPOS}
+
+check-env:
+ifndef RASRM_HOME
+	$(error RASRM_HOME environment variable is not set.)
+endif
+	@ printf "\n[RASRM_HOME set to ${RASRM_HOME}]\n"
+
+clone: check-env
+	@ printf "\n[Cloning into ${RASRM_HOME}]\n"
+	@ for r in ${REPOS}; do \
+		echo "($${r})"; \
+		if [ ! -e ${RASRM_HOME}/$${r} ]; then \
+			git clone git@github.com:ONSdigital/$${r}.git ${RASRM_HOME}/$${r}; \
+		else \
+			echo "  - already exists: skipping"; \
+		fi; echo ""; \
+	done
+
+build: clone
+	@ printf "\n[Building ${RASRM_HOME}]\n"
+	@ for r in ${REPOS}; do \
+		echo "Building ($${r})"; \
+		if [ ! -e ${RASRM_HOME}/$${r}/pom.xml ]; then \
+			if [ $${r} == "rm-comms-template-service" ]; then \
+				docker build -t sdcplatform/comms-template-svc ${RASRM_HOME}/$${r}; \
+			elif [ $${r} == "rm-survey-service" ]; then \
+				docker build -t sdcplatform/surveysvc ${RASRM_HOME}/$${r}; \
+			else \
+				docker build -t sdcplatform/$${r} ${RASRM_HOME}/$${r}; \
+			fi; echo ""; \
+		else \
+			mvn clean install -Dmaven.test.skip=true -DskipITs -f ${RASRM_HOME}/$${r}; \
```

Comment:
If we're skipping the IT tests, maybe a good shout to put `-DdockerComposeSkip -Ddockerfile.skip -Ddocker.skip` on there as well so it doesn't  build the docker images for the tests

Reply:
i still want it to build the dockerfile thou

Reply:
I didn't mean to copy `Ddockerfile.skip` 😆The docker compose should be able to be there because it's used to bring up test containers

Reply:
`-Dhttp.wait.skip` could be there as well to stop the tests from waiting to the containers to be healthy

Reply:
yeh i just added that while testing